src/server/pegasus_server_impl.cpp

```diff
@@ -3166,6 +3171,19 @@ void pegasus_server_impl::reset_usage_scenario_options(
     target_opts->max_write_buffer_number = base_opts.max_write_buffer_number;
 }
 
+void pegasus_server_impl::reset_allow_ingest_behind_option(
+    const rocksdb::DBOptions &base_db_opt,
+    rocksdb::DBOptions *target_db_opt,
+    const std::map<std::string, std::string> &envs)
+{
+    if (envs.empty()) {
+        // for reopen db during load balance learning
+        target_db_opt->allow_ingest_behind = base_db_opt.allow_ingest_behind;
+    } else {
+        target_db_opt->allow_ingest_behind = parse_allow_ingest_behind(envs);
+    }
```

Comment:
Is there any problem if just assign `base_db_opt` to `target_db_opt`, then update `target_db_opt` if `envs` is not empty? I guess there maybe some other configs are needed to load from the existing disk option file.

Reply:
When I read this slice of code,I think all of the db_option should copy to the new db after we reopen db. Because in my opinion,we only reopen db when replicas learn. And I think learnee's rocksDB should be total similar with learner's rocksDB. 
But I see the current logic only deal with the parameter `allow_ingest_behind`，and I'm not very sure that would be a special design for some feature?  As we know, `learn` this a very important part for pegasus,and some large feature base on it.